README.md

```diff
@@ -85,6 +87,7 @@ List of content:
 * [Turkey](https://t.me/Mina_Turkiye) 🇹🇷 Turkish telegram community
 * [France](https://t.me/mina_france) 🇫🇷 French telegram community
 * [Ukraine](https://t.me/mina_ukraine) 🇺🇦 Ukrainian telegram community
+* [India](https://t.me/Mina_india) IN Indian telegram community
```

Comment:
Please respect the styling of the entries:

`IN Indian telegram community` should become `🇮🇳 Indian telegram community`